src/index.html

```diff
@@ -4,24 +4,23 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css">
+  <!-- <link rel="stylesheet" href="./style.css"> -->
```

Comment:
- [ ] please try to remove the unused commented code in order to keep the file clean and professional